command/server/init.go

```diff
@@ -308,3 +315,56 @@ func (p *serverParams) initGRPCAddress() error {
 
 	return nil
 }
+
+// initBootnodeConfig initializes the bootnode configuration from various sources
+func (p *serverParams) initBootnodeConfig() error {
+	var err error
+	p.bootnodeConfig, err = p.getBootnodeConfig()
+	return err
+}
+
+// getBootnodeConfig retrieves bootnode configuration from various sources
+func (p *serverParams) getBootnodeConfig() (*chain.Bootnode, error) {
+	var bootnodes []string
+
+	// First try to load from specified bootnode path
+	if p.rawConfig.BootnodePath != "" {
+		// Load the bootnode configuration file directly
+		data, err := os.ReadFile(p.rawConfig.BootnodePath)
+		if err == nil {
```

Comment:
If there is an error here, it is silently missed, while the user expect a specific config to be loaded